Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector.yml

```diff
@@ -21,15 +21,25 @@ configuration:
   name: credentials
   displaypassword: Client Secret
   required: true
+  section: Connect
 - display: Export Profiles
-  additionalinfo: Internal document names or GUIDs of the Digital Guardian ARC export profiles.
+  additionalinfo: Internal document names or GUIDs of the Digital Guardian ARC export profiles. Custom export profiles are not officially supported.
   name: export_profile
   required: true
   type: 16  # Multi Select
   defaultvalue: defaultExportProfile
   options:
     - defaultExportProfile
-  section: Connect
+    - demisto
+  section: Collect
+- display: Number of Export Requests per Fetch
+  additionalinfo: Number of API calls per fetch to export events for each configured Digital Guardian ARC export profile. The value should be set to between 1 and 4.
```

Comment:
```suggestion
  additionalinfo: Number of API calls per fetch to export events for each configured Digital Guardian ARC export profile. Use with extreme caution as this might impact rate-limits and performance. Consult with engineering team before changing this value.
```

Reply:
Updated description based on suggestion (and synced changes to integration `README.md`)